src/compiletest/errors.rs

```diff
@@ -84,8 +124,9 @@ fn parse_expected(last_nonfollow_error: Option<usize>,
     let letters = line[kind_start..].chars();
     let kind = letters.skip_while(|c| c.is_whitespace())
                       .take_while(|c| !c.is_whitespace())
```

Comment:
Surely we have some higher-level way to extract the first word from the string by now? Can't we just call `split_whitespace()` or something?